Tiltfile

```diff
@@ -2,7 +2,7 @@
 
 BASE_IMAGE_MINIMAL="gcr.io/distroless/base"
 BASE_IMAGE_FULL="debian:bullseye-slim"
-BUILDER_IMAGE="golang:1.19-bullseye"
+BUILDER_IMAGE="golang:1.22.2-bullseye"
```

Comment:
Would the latest (i.e. `golang:1.22-bullseye` work?

Reply:
yes updated.